FastBank.Services/BankAccountService/BankAccountService.cs

```diff
@@ -119,7 +119,12 @@ public void TransferMoneyToFriend(Customer customer, BankAccount customerBankAcc
             if (emailFriend == "quit")
                 return;
 
-            var friend = friends.FirstOrDefault(f => f.Value.Email == emailFriend).Value;
+            var friend = friends.Where(f => f.Value.Email == emailFriend).Select(f => f.Value).FirstOrDefault();
```

Comment:
This is one way to solve the issue. 
The other way is:
`var friend = friends.FirstOrDefault(f => f.Value.Email == emailFriend)?.Value;`